functions/model/Product.js

```diff
@@ -0,0 +1,46 @@
+class Product {
+    constructor(product) {
+        if(product === undefined){
+            throw new Error('Send Product Details');
+        }
+        if(product.id === undefined){
+            throw new Error('Send Product Id');
+        }
+        
+        this.id = product.id;
+        this.name = product.name;
+        this.price = product.price;
```

Comment:
Do we need some checks here?
Like price must not be null or name must not be null?

Again Not Important